### PR TITLE
[4.4] Bug 1847457: Fix Dockerfiles

### PR DESCRIPTION
**Pull request by tnozicka**
backport of https://github.com/openshift/cluster-config-operator/pull/136

/cc @sttts

### CODE REVIEW

File: Dockerfile.rhel7 [hunk 1]
```diff
@@ -7,7 +7,7 @@ RUN GODEBUG=tls13=1 go build -ldflags "-X $GO_PACKAGE/pkg/version.versionFromGit
 FROM registry.svc.ci.openshift.org/ocp/4.2:base
 RUN mkdir -p /usr/share/bootkube/manifests/manifests
 RUN mkdir -p /usr/share/bootkube/manifests/bootstrap-manifests
-COPY --from=builder /go/src/github.com/openshift/cluster-config-operator/vendor/github.com/openshift/api/config/v1/*_config-operator_*-.yaml /usr/share/bootkube/manifests/manifests
+COPY --from=builder /go/src/github.com/openshift/cluster-config-operator/vendor/github.com/openshift/api/config/v1/*_config-operator_*.yaml /usr/share/bootkube/manifests/manifests
```
Comment (sttts) [inline Dockerfile.rhel7:10]: what does this mean for the 4.4 release?

Comment (tnozicka) [reply Dockerfile.rhel7:10]: could you be more specific?

**Comment by eparis** [comment]
worst case we go from being unable to build to being unable to build. So I'm letting it in.

**Pull request merged**